discovery_engine_core/core/test-fixtures/articles-some-malformed-media-urls.json

```diff
@@ -18,7 +18,7 @@
     ],
     "media": "",
     "is_opinion": false,
-    "twitter_account": "@XaynHQ",
+    "twitter_account": "@Example",
```

Comment:
:wrench: This is a valid twitter account. I guess we can use `xaynhq` here. I'm not sure if there is some handle reserved for this use case.

Reply:
well that is unexpected :O looks like the twitter api doesn't provide a dummy handle, i'll revert this change.